src/client/utils.test.ts

```diff
@@ -42,6 +48,20 @@ describe('replaceUrlParams', () => {
   })
 })
 
+describe('replaceUrlProtocol', () => {
+  it('Should replace http to ws', () => {
+    const url = 'http://localhost'
+    const newUrl = replaceUrlProtocol(url, 'ws')
+    expect(newUrl).toBe('ws://localhost')
+  })
+
```

Comment:
Can you write a test for the case if the url is `https://localhost`?

Reply:
ok!

Reply:
Thanks for the review.

In addition to https://localhost, I added a test for wss://localhost! https://github.com/honojs/hono/pull/2479/commits/0bffae5199da73298845472ad2b58992c4364d71